spaces/S000183/README.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: S000183
+name: Non-sequentially discrete countable complement topology
+aliases:
+  - Modified cocountable space
```

Comment:
Hmm, how about removing the alias and merging it into the name, something like: 
"Non-sequentially discrete modified cocountable topology"?

Reply:
I'll make a modification for this and https://topology.pi-base.org/spaces/S000017 to line them up a bit.

Reply:
I like the new name for S17 too.